client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-how-to-migrate/index.tsx

```diff
@@ -1,8 +1,9 @@
-import { Card } from '@automattic/components';
-import { StepContainer, SubTitle, Title } from '@automattic/onboarding';
+import { StepContainer } from '@automattic/onboarding';
+import { __experimentalVStack as VStack } from '@wordpress/components';
```

Comment:
I think we should not use the experiment component since the Gutenberg team can easily break the API and we will have zero visibility about it since we don't have automated tests.

Reply:
Agreed - it feels like doing this using more basic CSS will be more robust in the long term.